libs/components/forms/src/lib/modules/file-attachment/file-drop.component.spec.ts

```diff
@@ -29,7 +34,11 @@ describe('File drop component', () => {
 
   beforeEach(() => {
     TestBed.configureTestingModule({
-      imports: [SkyFileAttachmentsModule],
+      imports: [
+        SkyFileAttachmentsModule,
+        SkyHelpTestingModule,
+        NoopAnimationsModule,
```

Comment:
If popovers needs `NoopAnimationsModule`, I would put it in the `exports` of `SkyHelpTestingModule` so that we don't have to import `NoopAnimationsModule` every time we want to test help popovers.

Reply:
checkbox and checkbox group unit tests didnt actually end up needing `NoopAnimationsModule` but the file drop service did. I am not entirely sure why it affects here and not in the other PR but i did go ahead and address this comment in this PR bc of that.